nzedb/Binaries.php

```diff
@@ -724,27 +748,33 @@ public function scan($groupMySQL, $first, $last, $type = 'update', $missingParts
 					// Get the current unixtime from PHP.
 					$now = time();
 
+					$xref = ($multiGroup === true ? sprintf('xref = CONCAT(xref, "\\n"%s ),', $this->_pdo->escapeString(substr($header['Xref'], 2, 255))) : '');
 					$collectionID = $this->_pdo->queryInsert(
 						sprintf("
 							INSERT INTO %s (subject, fromname, date, xref, group_id,
 								totalfiles, collectionhash, dateadded)
 							VALUES (%s, %s, FROM_UNIXTIME(%s), %s, %d, %d, '%s', NOW())
-							ON DUPLICATE KEY UPDATE dateadded = NOW(), noise = '%s'",
-							$tableNames['cname'],
+							ON DUPLICATE KEY UPDATE %s dateadded = NOW(), noise = '%s'",
+							($multiGroup === true ? 'mgr_collections' : $tableNames['cname']),
```

Comment:
Style: Ternaries should always be on their own line, never part of a more complex statement.